tony-core/src/main/java/com/linkedin/tony/TonyClient.java

```diff
@@ -853,6 +853,7 @@ public boolean monitorApplication() throws YarnException, IOException, Interrupt
         break;
       }
 
+<<<<<<< HEAD
```

Comment:
Can you resolve the merge conflicts? There shouldn't be any "<<<...HEAD", "===", or ">>>...." (this line, line 866, line 877).